webservices/resources/totals.py

```diff
@@ -6,19 +6,23 @@
 from webservices import utils
 from webservices import schemas
 from webservices.common import models
-from webservices.common.models import db, TransactionCoverage
 from webservices.common.views import ApiResource
 from webservices.utils import use_kwargs
-from webservices.resources.reports import reports_type_map
 
+committee_type_map = {
+    'house-senate': 'H',
+    'presidential': 'P',
+    'ie-only': 'I',
+    'pac-party': None,
+    'pac': 'O',
+    'party': 'X'
+}
 
```

Comment:
house and senate are separate committee types.  It's better to separate it to avoid future potential confusion
house : H
senate: S

"pac-party" is not a valid committee type.  Please confirm @jwchumley @PaulClark2